portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,134 @@
+                            </div>  
+                        </div>
+
+                        <div class="sect-5">
+                            <div id="sect-5">
+                                <p>Playing the guitar.</p>
+                                <p><button onclick="addRandomSong()">My Favorite Songs to Play</button>
+                                <div id="song-container" class="rainbow-text"></div></p>
+                    
+                                <p>Musicals.</p>
+                                <p><button onclick="addRandomMusical()">My Favorite Musicals</button>
+                                <div id="musical-container" class="rainbow-text"></div></p>
+                            </div>
+                        </div>
+
+                        <div class="sect-6">
+                            <div id="sect-6">
+                                <p>Making art.</p>
+                                <p><button onclick="document.location = 'artimages.html'" 
+                                           class="rainbow-text">Some of my Art</button></p>
+                                <p>Hiking, exploring, looking at, and traveling the world. </p>
+                                <p><button onclick="document.location = 'images.html'" 
+                                           class="rainbow-text">My Happy Places</button></p>
+                            </div> 
+                        </div>
+
+                        <div class = "more rainbow-text">
+                            <div id = "section-title">
+                                <h2>Want to learn more?</h2>
+                            </div>
+                        </div>
+
+                        <div class="sect-7">
+                            <div id = "sect-7">
+                                <p>Check out my:</p>
+                                <a href = "https://www.linkedin.com/in/abbymapes">
+                                <img src = "/images/linkedin.png" alt = "LinkedIn Logo." 
+                                    width = "100 px"/></a>
+                                <br>
+                                <a href = "https://github.com/abbymapes">
+                                <img src = "/images/git.png" alt = "GitHub Logo." 
+                                    width = "100 px"/></a>
+                                <br>
+                            </div>
+                        </div>
+
+                        <div class="goodbye">
+                            <p>I hope you enjoyed learning a little more about 
+                            me! Sadly, it is time to say:</p>
+                            <p><button onclick="addRandomGoodbye()">Click me!</button>
+                            <div id="goodbye-container" class="rainbow-text"></div></p>
+                            <br>
+                        </div>
+                    </div>
+
+                    <div class="bottom rainbow-text">
+                        <h1> </h1>
```

Comment:
did you mean to leave this tag empty?

Reply:
Yes. It gave the footer more height. I can remove it and adjust the height if you recommend doing so.